admin/app/components/solidus_admin/orders/index/component.yml

```diff
@@ -0,0 +1,13 @@
+# Add your component translations here.
+# Use the translation in the example in your template with `t(".hello")`.
+en:
+  create_order: 'Create Order'
+  give_feedback: 'Give feedback'
+  feedback_description: 'We are constantly trying to improve. Please let us know what you think about this admin page.'
```

Comment:
Same idea about extracting it into a feedback component.

Reply:
Done 👍 https://github.com/solidusio/solidus/pull/5347#discussion_r1303073213